db/migrate/20160225130400_drop_content_owners.rb

```diff
@@ -0,0 +1,61 @@
+class DropContentOwners < ActiveRecord::Migration
```

Comment:
It'd be worse I think. It's got if conditions in it. It was never going to be a thing of beauty..